module/Api/src/Domain/CommandHandler/Tm/UpdateNysiisName.php

```diff
@@ -41,24 +41,8 @@ public function handleCommand(CommandInterface $command)
     {
         $this->result->addMessage('TM NYSIIS currently disabled');
         return $this->result;
-
```

Comment:
Possibly one for @sdh100shaun to answer, but is Nysiis definitely never coming back? If it might then we need to keep this code, if it isn't then we can delete a lot more...

Reply:
It can be retrieved from the Git history if we need it again.